src/main/java/duke/command/MarkCommand.java

```diff
@@ -0,0 +1,45 @@
+package duke.command;
+
+import duke.Storage;
+import duke.Ui;
+import duke.task.TaskList;
+
+/**
+ * Represents the command for marking a task in Duke's TaskList.
+ */
+public class MarkCommand implements Command{
+    private final int toMark;
```

Comment:
Similarly, I think `toMark` should be capitalised since it is a constant? (There might be a few other instances of this styling error in other parts of the code)